openmdao/core/tests/test_parallel_derivatives.py

```diff
@@ -825,8 +825,6 @@ def setup(self):
                 self.add_input('x', shape=size)
                 self.add_output('y', shape=size)
                 self.add_output('y2', shape=size)
-                self.declare_partials('y', 'x')
```

Comment:
I assume that, if we want to get the most efficiently total model coloring, we still need to declare the partials (maybe a component with multiple outputs where each output depends on a subset of the inputs.)  Do we still have a test in our suite where we do that?

Reply:
Turns out we don't support computing total model coloring when there are matrix-free comps, so I withdraw this comment.